examples/basic/config.json

```diff
@@ -1,103 +1,159 @@
+        },
+        {
+          "destination": "StyleDictionaryColor.m",
+          "format": "ios/colors.m",
+          "className": "StyleDictionaryColor",
+          "type": "StyleDictionaryColorName",
+          "filter": {
+            "attributes": {
+              "category": "color"
+            }
           }
-        }
-      },{
-        "destination": "StyleDictionarySize.h",
-        "format": "ios/static.h",
-        "className": "StyleDictionarySize",
-        "type": "float",
-        "filter": {
-          "attributes": {
-            "category": "size"
+        },
+        {
+          "destination": "StyleDictionarySize.h",
+          "format": "ios/static.h",
+          "className": "StyleDictionarySize",
+          "type": "float",
+          "filter": {
+            "attributes": {
+              "category": "size"
+            }
           }
-        }
-      },{
-        "destination": "StyleDictionarySize.m",
-        "format": "ios/static.m",
-        "className": "StyleDictionarySize",
-        "type": "float",
-        "filter": {
-          "attributes": {
-            "category": "size"
+        },
+        {
+          "destination": "StyleDictionarySize.m",
+          "format": "ios/static.m",
+          "className": "StyleDictionarySize",
+          "type": "float",
+          "filter": {
+            "attributes": {
+              "category": "size"
+            }
           }
         }
-      }]
+      ]
     },
     "ios-swift": {
       "transformGroup": "ios-swift",
       "buildPath": "build/ios-swift/",
-      "files": [{
-        "destination": "StyleDictionary.swift",
-        "format": "ios-swift/class.swift",
-        "className": "StyleDictionary",
-        "filter": {}
-      }]
+      "files": [
+        {
+          "destination": "StyleDictionary.swift",
+          "format": "ios-swift/class.swift",
+          "className": "StyleDictionary",
+          "filter": {}
+        }
+      ]
     },
     "ios-swift-separate-enums": {
       "transformGroup": "ios-swift-separate",
       "buildPath": "build/ios-swift/",
-      "files": [{
-        "destination": "StyleDictionaryColor.swift",
-        "format": "ios-swift/enum.swift",
-        "className": "StyleDictionaryColor",
-        "filter": {
-          "attributes": {
-            "category": "color"
+      "files": [
+        {
+          "destination": "StyleDictionaryColor.swift",
+          "format": "ios-swift/enum.swift",
+          "className": "StyleDictionaryColor",
+          "filter": {
+            "attributes": {
+              "category": "color"
+            }
           }
+        },
+        {
+          "destination": "StyleDictionarySize.swift",
+          "format": "ios-swift/enum.swift",
+          "className": "StyleDictionarySize",
+          "type": "float",
+          "filter": {
+            "attributes": {
+              "category": "size"
+            }
+          }
+        }
+      ]
+    },
+    "flutter": {
```

Comment:
Can you create this in its own example, rather than add it to the basic one?

Reply:
You mean like a Flutter specific example? Where it should be located in the examples directory?

Reply:
Yes, I think it would make sense to add it in the 'advanced' directory of examples. You can follow the other examples in there

Reply:
Moved to advanced/flutter